packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -274,6 +274,49 @@ export interface RaiseHandCallFeatureState {
   localParticipantRaisedHand?: RaisedHandState;
 }
 
+/* @conditional-compile-remove(together-mode) */
+/**
+ * State only version of {@link @azure/communication-calling#TogetherModeCallFeature}. {@link StatefulCallClient} will
+ * automatically listen for raised hands on the call and update the state exposed by {@link StatefulCallClient} accordingly.
+ * @alpha
+ */
+export interface TogetherModeCallFeatureState {
+  /**
+   * Proxy of {@link @azure/communication-calling#TogetherModeCallFeature.togetherModeStream}.
+   */
+  stream?: TogetherModeStreamState;
+}
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * State only version of {@link @azure/communication-calling#TogetherModeVideoStream}.
+ * @alpha
+ */
+export interface TogetherModeStreamState {
+  /**
+   * Proxy of {@link @azure/communication-calling#TogetherModeVideoStream.id}.
+   */
+  id: number;
+  /**
+   * Proxy of {@link @azure/communication-calling#TogetherModeVideoStream.mediaStreamType}.
+   */
+  mediaStreamType: MediaStreamType;
```

Comment:
Can together mode be audio or screenshare?
![image](https://github.com/user-attachments/assets/371b405f-ebf0-4f1c-853f-7835b6277392)

Reply:
no. The stream type is video